app/src/main/java/org/wikipedia/readinglist/database/ReadingListDbHelper.java

```diff
@@ -74,6 +75,16 @@ public static ReadingListDbHelper instance() {
         return lists;
     }
 
+    private ReadingList getListWithoutContentsById(SQLiteDatabase db, long readingListId) {
```

Comment:
Return value should probably be annotated `@Nullable`.

Reply:
Done!